src/wallet/rpcdump.cpp

```diff
@@ -876,6 +877,16 @@ static UniValue ProcessImport(CWallet * const pwallet, const UniValue& data, con
             throw JSONRPCError(RPC_INVALID_ADDRESS_OR_KEY, "Invalid redeem script");
         }
 
+        // Add to keypool only works with privkeys disabled
+        if (add_keypool && !pwallet->IsWalletFlagSet(WALLET_FLAG_DISABLE_PRIVATE_KEYS)) {
+            throw JSONRPCError(RPC_INVALID_PARAMETER, "Keys can only be imported to the keypool when private keys are disabled");
+        }
+
+        // Add to keypool only works with pubkeys
+        if (add_keypool && pubKeys.size() == 0) {
+            throw JSONRPCError(RPC_INVALID_PARAMETER, "Only pubkeys be imported to the keypool");
```

Comment:
Commit "Add option to importmulti add an imported pubkey to the keypool"

... can be ...

Reply:
Fixed